src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -239,6 +240,15 @@ public class LivingEntityGUI extends ModElementGUI<LivingEntity> implements IBlo
 	private final JCheckBox spawnThisMob = new JCheckBox();
 	private final JCheckBox doesDespawnWhenIdle = new JCheckBox();
 
+	private final JSpinner[] spawnPercentageRaid = new JSpinner[] {

```

Comment:
![image](https://github.com/MCreator/MCreator/assets/16374228/4c41701b-8944-43d2-83b9-2b323941d3c7)

There is no location here where it would explain how to make this field enabled - what is needed for this to work.

It should also be noted that this is specific parameter, so put it below dungeon spawning parameter, not between some of the most important spawning parameters.

Reply:
Fixed